src/actions/featureActions.js

```diff
@@ -0,0 +1,94 @@
+import {
+  SET_INIT_SLIDE,
+  SET_NEXT_SLIDE,
+  SET_PREVIOUS_SLIDE,
+} from './types';
+
+
+/**
+ * @param {function} payload
+ *  @returns {Object} action obj
+ */
+export function initSlide(payload) {
+  return { type: SET_INIT_SLIDE, payload };
+}
+
+/**
+ * @param {function} payload
+ *  @returns {Object} action obj
+ */
+export function nextSlide(payload) {
+  return { type: SET_NEXT_SLIDE, payload };
+}
+
+/**
+ * @param {function} payload
+ *  @returns {Object} action obj
+ */
+export function previousSlide(payload) {
+  return { type: SET_PREVIOUS_SLIDE, payload };
+}
+
+
+export const setInitData = () => {
```

Comment:
Why do we need this?

Reply:
There is going to be an asynchronous call in that method in the future. The data passed to it now is static and it allows me the option of calling different action creators in the method. The reason I say 'future' is because the API for featured arts has not been implemented